### PR TITLE
Chore merge main from 2d0c94c to a6d8aa8

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/2d0c94c05a8d338b1644a49820747d0cf72621ad
- [x] https://github.com/twbs/bootstrap/commit/6855ce9512d0cd0b4c54f720ac5b9a15c685e057 
- [x] https://github.com/twbs/bootstrap/commit/f52d4a35b3332c34615880c6f6e8a026363ca390
- ~~https://github.com/twbs/bootstrap/commit/5d4048350f000710d228ee341a093784ca0ae988~~: most of the examples are not in Boosted, and for the grid one, I have no idea what are the tools used to reduce the size here, so IMO nothing to do for this commit
- [x] https://github.com/twbs/bootstrap/commit/a6d8aa827d645796cf986e289b7311360318a21f

### CODE REVIEW

File: site/content/docs/5.3/utilities/object-fit.md [hunk 1]
```diff
@@ -25,23 +25,23 @@ Classes for the value of `object-fit` are named using the format `.object-fit-{v
...
-{{< placeholder width="140" height="80" class="object-fit-lg-contain border" text="Contain on lg" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-xl-contain border" text="Contain on xl" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-xxl-contain border" text="Contain on xxl" markup="img" >}}
+{{< placeholder width="140" height="80" class="object-fit-sm-contain border" text="Contain on sm" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-md-contain border" text="Contain on md" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-lg-contain border" text="Contain on lg" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-xl-contain border" text="Contain on xl" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-xxl-contain border" text="Contain on xxl" markup="img" color="#868e96" background="#dee2e6" >}}
```
Comment (louismaximepiton) [inline site/content/docs/5.3/utilities/object-fit.md:44]: Should we try to use Orange color in here ? For instance, `#999999` or `#666666` and `#eeeeee` or `#dddddd`

**Review by louismaximepiton — COMMENTED** [review]
One question and we're good to go

File: site/content/docs/5.3/utilities/object-fit.md (hunk 1 above)
Comment (julien-deramond) [reply site/content/docs/5.3/utilities/object-fit.md]: Changed it via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1907/commits/3e7653e0d20b810bcd0c36b62be9b2ac6c69a37a

**Review by louismaximepiton — APPROVED** [review]
👌

**Pull request merged**